crates/cli/src/utils/deployments.rs

```diff
@@ -387,18 +388,17 @@ pub async fn create_deployment(
 
     trpc_client.client.request(request).await?;
 
-    // TODO upload in parallel
+    let mut join_set = tokio::task::JoinSet::new();
     for (asset, url) in assets_urls {
         let asset = assets
             .get(&asset)
             .unwrap_or_else(|| panic!("Couldn't find asset {asset}"));
 
-        let request = Request::builder()
-            .method(Method::PUT)
-            .uri(url)
-            .body(Body::from(asset.clone()))?;
+        join_set.spawn(upload_asset(trpc_client.clone(), asset.clone(), url));
+    }
 
-        trpc_client.client.request(request).await?;
+    while let Some(res) = join_set.join_next().await {
+        res?.expect("Couldn't upload asset {asset} to url {url}");
```

Comment:
`{asset}` and `{url}` aren't interpreted here, it's ok to only keep the beginning:

```suggestion
        res?.expect("Couldn't upload asset");
```